frontend/src/pages/ApplicantTableView/PositionCard.tsx

```diff
@@ -0,0 +1,51 @@
+import { Chip, Paper, Stack, Typography, Box } from "@mui/material";
```

Comment:
My guess is that this whole file is code that exists elsewhere, for example in JobFeed and the other referral views. Worth looking into and refactoring to have just once